test/runtime_tests.cc

```diff
@@ -230,7 +230,7 @@ namespace detail {
 	}
 
 	TEST_CASE("task_manager invokes callback upon task creation", "[task_manager]") {
-		task_manager tm{1, nullptr};
+		task_manager tm{1, nullptr, {}};
```

Comment:
Since this is no longer a `std::optional` I'd suggest to change this for consistency (here and below).
```suggestion
		task_manager tm{1, nullptr, nullptr};
```

Reply:
Yep.